packages/wallet/README.md

```diff
@@ -0,0 +1,39 @@
+# Wallet Management JS module
+This is a wallet management JavaScript module for Stellar and Ethereum.
+## Getting Started
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
+### Prerequisites
+* [Node](https://nodejs.org/en/download/)
+* [NPM](https://www.npmjs.com/), installed by the command `npm install npm@latest -g`.
+### Installing
+1. Clone or download the repo.
+2. Run `npm install` in this directory in the command line.
+## Running the tests
+* Tests are wirtten using [Jest](https://jestjs.io/) library.
+* Run `npm test` to run all the tests.
+* You can configure testing in the `package.json` file, or by adding arguments to the command.
+* E.g. change *line 7* in `package.json` to `"test": "jest --coverage WalletManager.test.js"`.
+* Test files include `Account.test.js`, `Account.async.test.js`, `Account.anchor.test.js`, `WalletManager.test.js`
+* From the test cases you can see how to use this package.
```

Comment:
Let's have a small section to show a quick code example of what the package does 🙂

Reply:
updated

Reply:
code block can be ```js to preview as javascript code